### DISCUSSION TITLE
部署完成后无法通过域名无法访问，页面显示nginx404

### DISCUSSION THREAD

**Discussion by aodebiu** [labring/sealos#4146] [Q&A] [answered]
master0：192.168.1.200
node0：192.168.1.201
访问电脑主机：192.168.1.199
域名：www.example.com
安装命令：curl -sfL https://raw.githubusercontent.com/labring/sealos/main/scripts/cloud/install.sh -o /tmp/install.sh && bash /tmp/install.sh 
操作：
1、中英文选择：1
2、master0节点：192.168.1.200
3、node0节点：192.168.1.201
4、ssh免密已配置
5、cloudDomain：www.example.com
6、端口10443
7、证书：n
安装完成后访问生成的地址：www.example.com显示443，请问是哪里有问题？

**Comment by bxy4543** [comment]
cloudDomain 得写你自己的域名，并且指定证书，没有的话就用nip.io，也就是填入 192.168.1.200.nip.io，然后就可以访问192.168.1.200.nip.io:10443 了

**Reply by aodebiu** [reply]
![image](https://github.com/labring/sealos/assets/44352658/637a3d1f-9cc8-498d-90d9-713595812705)
目前按您说的我重新部署了一遍还是没有办法访问，我不是书签发证书选了n，是不是和这个有关系，还是说我需要在hosts中加入这个指定的解析

**Reply by bxy4543** [reply, marked as answer]
没关系，自定义端口之前确实存在问题 ，在这里 #4271 修复
